oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1188,6 +1188,9 @@ void pldm::responder::oem_ibm_platform::Handler::upadteOemDbusPaths(
 void pldm::responder::oem_ibm_platform::Handler::_processSystemReboot(
     sdeventplus::source::EventBase& /*source */)
 {
+    BiosAttributeList biosAttrList;
```

Comment:
So just to double check, the only time we go into this _processSystemReboot() function is when we are going through the inband update path? Or really, we only go through this path when the host is responsible for asking for the BMC reboot and re-ipl after that?